src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -102,6 +108,9 @@ def load_arguments(self, _):
                    deprecate_info=c.deprecate(target='az spring-cloud update --disable-app-insights',
                                               redirect='az spring-cloud app-insights update --disable',
                                               hide=True))
+        c.argument('build_pool_size',
```

Comment:
add preview = true and note this is (enterprise only)

Reply:
OK, added.